ouroboros-consensus/docs/.dir-locals.el

```diff
@@ -0,0 +1,5 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+
+((nil
+  (fill-column . 80)))
```

Comment:
I was expecting `latex-mode` instead of `nil`, but indeed, `nil` is good if not better.

Thought: why not promote it up one directory level so that it also applies to the rest of the consensus code, i.e., the comments?

Reply:
I wondered the same, so I've done so now.